backend/supabase/snippets/Untitled query 230.sql

```
@@ -0,0 +1,45 @@
+insert into public.vendor_achievements
+  (vendor_id, code, title, description, icon, tier)
+select
+  id,
+  badge.code,
+  badge.title,
+  badge.description,
+  badge.icon,
+  badge.tier
+from public.vendors
```

SQLFluff (4.2.2)

[error] Duplicate table alias 'vendors'. Table aliases should be unique.

(AL04)